web/src/components/App.tsx

```diff
@@ -65,18 +58,14 @@ class App extends React.Component<Props, State> {
     }
 
     if (RemoteData.isSuccess(errors)) {
-      let decodedErrorKey = decodeURIComponent(match.params.errorKey)
-      let activeError = errors.data.find(e => e.aggregation_key === decodedErrorKey)
-      if (
-        activeError !== undefined &&
-        (activeError !== decodedErrorKey) &&
-        !RemoteData.isLoading(errors)) {
-        setActiveError(activeError.aggregation_key)
-      }
+      const decodedErrorKey = decodeURIComponent(match.params.errorKey)
+      const activeError = errors.data.find(e => e.aggregation_key === decodedErrorKey)
+
+      if (!activeError || RemoteData.isLoading(errors)) return
```

Comment:
Is this `return` statement expected?

Reply:
Yes, my intention with this return is to escape the if as early as possible to avoid the if-else statements.

Reply:
Same here, I was a bit confused by the syntax :) I _think_ that the `RemoteData.isLoading` call is not necessary because we're already in a branch where `RemoteData.isSuccess` is true but I also saw that we used it before. Doesn't look like we need to worry about this now.